### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nadavsu**
CONTRIBUTING.md explains new-comers the guidelines of contribution to the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Contributing to roo.me
+
+Thank you for supporting too.me and looking for ways to help. Please note that some conventions here might be a bit different than what you are used to, even if you have contributed to other open source projects before. Reading this document will help you save time and work  effectively with the developers and other contributors.
```
Comment (AmitAharoni) [inline CONTRIBUTING.md]: line 3 - 'r'oo.me instead of  't'oo.me

Comment (nadavsu) [reply CONTRIBUTING.md]: Good catch.

**Review by DanielMy777 — COMMENTED** [review]
Super nice addition!

**Review by Omeramsc — APPROVED** [review]
LGTM
You can add a few words about setting a local environment, and about the CI.

**Review by Micha546 — COMMENTED** [review]
Nice! 🔥

**Comment by OrAsaf** [comment]
LGTM

**Pull request merged**